pkg/app/mysql.go

```diff
@@ -57,6 +57,7 @@ func NewMysqlDB(name string) HelmApp {
 				"auth.rootPassword": "mysecretpassword",
 				"image.pullPolicy":  "Always",
 			},
+			Version: "12.2.4",
```

Comment:
Ideally we should update/fix the BP to work with the latest MySQL if that is failing

Reply:
It's failing during installation itself with Access denied error.